types/policy_sync.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+type PolicySyncStatus int
+
+const (
+	NotSynced PolicySyncStatus = iota
+	Synced    PolicySyncStatus = iota + 1
+	Failed    PolicySyncStatus = iota + 2
+)
+
+type PluginPolicySync struct {
+	PolicyID   string           `json:"policy_id" validate:"required"`
+	Status     PolicySyncStatus `json:"status" validate:"required"`
+	FailReason string           `json:"fail_reason"` // when synced is false, this field contains the reason for the failure
```

Comment:
I think we may need a version here

If a plugin server is down, it is possible that a user creates a policy, and edits it subsequently, before the first version has been synced

We should track the sync so we can ensure we don't sync stale policies (or at least do so sequentially)